content/templates.xqm

```diff
@@ -136,21 +138,26 @@ declare function templates:process($nodes as node()*, $model as map(*)) {
                 return
                     if ($dataAttr) then
                         templates:call($dataAttr, $node, $model)
-                    else
+                    else if (($model($templates:CONFIGURATION)($templates:CONFIG_USE_CLASS_SYNTAX), $templates:SEARCH_IN_CLASS)[1]) then
```

Comment:
`$templates:CONFIG_USE_CLASS_SYNTAX` should be set to `true()` in default configuration.
This will make the condition here much simpler as we can be sure this key is always set.

Reply:
```
else if ($model?($templates:CONFIGURATION)?($templates:CONFIG_USE_CLASS_SYNTAX)) then

Reply:
Implemented in the suggested way. See https://github.com/eXist-db/templating/pull/25/commits/0a9b651f4d7a15ee32584f7acfe3b8499afc2e1e